src/utils/getSortedGoods.ts

```diff
@@ -0,0 +1,36 @@
+import { SortType } from '../types/SortType';
+
+interface SortingFilters {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+export function getSortedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortingFilters,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods = preparedGoods.sort(
+      (currentGood: string, nextGood: string) => {
+        switch (sortField) {
+          case SortType.ALPHABETICALLY:
+            return currentGood.localeCompare(nextGood);
+
+          case SortType.LENGTH:
+            return currentGood.length - nextGood.length;
+
+          default:
+            return 0;
+        }
+      },
+    );
+  }
+
+  if (isReversed) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  return preparedGoods;
```

Comment:
The utility function itself doesn't directly interact with the DOM or React components, so make sure that when you use this function within a React component, you follow React best practices for state and props to trigger re-renders after sorting.